apps/marketing/src/components/(marketing)/widget.tsx

```diff
@@ -124,7 +124,7 @@ export const Widget = ({ className, children, ...props }: WidgetProps) => {
     setValue('signatureDataUrl', draftSignatureDataUrl);
     setValue('signatureText', '');
 
-    trigger('signatureDataUrl');
+    void trigger('signatureDataUrl');
```

Comment:
If I am not wrong, this resolves to `undefined` right?

Reply:
Yep we’re void’ing promises that we’re happy to leave floating